code/modules/mining/equipment/regenerative_core.dm

```diff
@@ -87,6 +87,13 @@
 			SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "core", /datum/mood_event/healsbadman) //Now THIS is a miner buff (fixed - nerf)
 			qdel(src)
 
+/obj/item/organ/regenerative_core/afterattack(atom/target, mob/user, proximity_flag=null)
+	. = ..()
+	applyto(target, user, proximity_flag)
```

Comment:
Doesn't make a lot of sense to send the proximity_flag down. Just handle that interaction in this proc.